metabase/metabase
# Add initial bulk filter modal

Epic https://github.com/metabase/metabase/issues/22629

Changes:
- Adds a modal to apply changes to multiple filters at once
- Adds `ButtonGroup`
- Adds `Tab` components. The API and implementation is the same as in material UI.

How to test:
- Clicking on `...` near the `Filter` button should open the modal with all available filter fields.
- Make sure it is possible to add a new filter, modify existing one, and remove one

Known issues:
- Some nested popover rendering issues, e.g. for dates filters
- It's only possible to add one new filter per field. If there are multiple filters in the query, it would only be possible to edit them. That's by design, but still mentioning it here.

<img width="1186" alt="Screenshot 2022-05-16 at 14 06 45" src="https://user-images.githubusercontent.com/8542534/168579766-d4bfe3f6-483d-49cd-bfe0-6713731b6550.png">
